scripts/build/restore-packages.ps1

```diff
@@ -17,13 +17,14 @@ else {
 }
 
 # Restore packages
+# NOTE(anurse): I had to remove --quiet, because NuGet3 is too quiet when that's provided :(
 header "Restoring packages"
-& "$DnxRoot\dnu" restore "$RepoRoot\src" --quiet --runtime "$Rid" "$NoCacheArg"
-& "$DnxRoot\dnu" restore "$RepoRoot\test" --quiet --runtime "$Rid" "$NoCacheArg"
-& "$DnxRoot\dnu" restore "$RepoRoot\tools" --quiet --runtime "$Rid" "$NoCacheArg"
+dotnet restore "$RepoRoot\src" --runtime "$Rid"
+dotnet restore "$RepoRoot\test" --runtime "$Rid"
+dotnet restore "$RepoRoot\tools" --runtime "$Rid"
```

Comment:
NuGet3 doesn't have a `--no-cache` option. Instead, we'll want to clear the cache before each build. Having said that, using the cache for now should be fine.